ts/Dashboard/README.md

```diff
@@ -0,0 +1,119 @@
+# Highcharts Dashboards #
+A JavaScript library for interactive dashboards.
+
+## Installation
+There are multiple different ways how you can import this package.
+Among the most popular are:
+* Using npm
+```bash
+    npm install <PACKAGE-NAME>
+```
+* Importing as a script
+```html
+    <script src="https://code.highcharts.com/dashboards.js"></script>
+ ```
+
+To fully utilize the Dashboards potential, there might be a need to load additional modules. <br>
+In the [documentation](https://highcharts.com/docs/dashboards/instalation), you can find more information on how to do that and what are the other ways of importing the Dashboards.
+
+## Components
+Each dashboard is built with different components. You can add the most basic HTMLComponent where you can add some text, image etc.
```

Comment:
Can we wrap names like HTMLComponent into `HTMLComponent`? It will increase readibility